src/v3/plugins/github/nodes.js

```diff
@@ -90,6 +90,9 @@ export type ParentAddress =
   | PullAddress
   | ReviewAddress;
 
+// Each of these types may have Comments as children
+export type CommentableAddress = IssueAddress | PullAddress | ReviewAddress;
```

Comment:
Should we check statically that `CommentableAddress <: ParentAddress`?

(I vote yes!)